src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/Utilities/ParsedDocument.cs

```diff
@@ -21,16 +21,12 @@ namespace Microsoft.CodeAnalysis;
 /// In exceptional cases such API may be asynchronous as long as it completes synchronously in most common cases and async completion is rare. It is still desirable to improve the design
 /// of such feature to either not be invoked on a UI thread or be entirely synchronous.
 /// </remarks>
-internal readonly record struct ParsedDocument(DocumentId Id, SourceText Text, SyntaxNode Root, HostLanguageServices LanguageServices)
+internal readonly record struct ParsedDocument(DocumentId Id, SourceText Text, SyntaxNode Root, HostLanguageServices HostLanguageServices)
```

Comment:
>HostLanguageServices [](http://example.com/codeflow?start=96&length=20)

For follow up: I'd expect this to be just LanguageServices. Is Workspace being accessed from these services?

Reply:
yup  agreed

Reply:
Have noted this for follow-up